cmd/init/presetdownloader/downloader.go

```diff
@@ -111,7 +111,7 @@ func DownloadPresetPipelines(ctx context.Context, repo repository.Repository) er
 	}
 
 	clientConn, err := grpc.NewClient(fmt.Sprintf("%s:%d", config.Config.InstillCloud.Host, config.Config.InstillCloud.Port),
-		grpc.WithTransportCredentials(credentials.NewTLS((&tls.Config{}))),
+		grpc.WithTransportCredentials(credentials.NewTLS((&tls.Config{MinVersion: tls.VersionTLS12}))),
```

Comment:
Nothing against being explicit about the minimum version but this is also the default value, so I'd be fine with `&tls.Config{}`, too.

Reply:
The default min version would have security issue, that's why I need to assign a specific version.

Reply:
But you assigned the same version as the [default min version](https://pkg.go.dev/crypto/tls#Config) 🤔 
```
	// By default, TLS 1.2 is currently used as the minimum. TLS 1.0 is the
	// minimum supported by this package.
```

Reply:
You're right. I've checked the Go 1.22.5 (the go.mod using this version) and 1.23, doc did mention the same thing. 
It seems `gosec`'s issue. If you don't set the MinVersion explicitly with specific version, it would keep warning you.
You want keep this change ? Or I can rollback

Reply:
It's fine as it is, being explicit about these things never hurts 👍